acceptance/gossip_peerings_test.go

```diff
@@ -45,11 +44,6 @@ func checkGossip(t *testing.T, l *localcluster.Cluster, d time.Duration,
 		case <-stopper:
 			t.Fatalf("interrupted")
 			return nil
-		case e := <-l.Events:
-			if log.V(1) {
-				log.Infof("%+v", e)
-			}
-			return fmt.Errorf("event: %+v", e)
```

Comment:
so you would keep AssertAndStop but only use if if nothing is expected, and then have boilerplate in each test that does non-trivial things? In an earlier comment you sounded like you'd rather avoid boilerplate in each test. My feeling is that whatever the interim solution for now is, it's going to be imperfect.